.gitignore

```diff
@@ -12,4 +12,4 @@
 *.out
 
 # Dependency directories (remove the comment below to include it)
-# vendor/
+vendor/
```

Comment:
We shouldn't be using vendor, so we probably don't need to add it to gitignore.

Reply:
:+1: